implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java

```diff
@@ -191,13 +191,44 @@
         return getValue(name, requireConverter(aClass));
     }
 
+    /**
+     * Return the content of the direct sub properties as the requested type of Map.
+     *
+     * @param name The configuration property name
+     * @param kClass the type into which the keys should be converted
+     * @param vClass the type into which the values should be converted
+     * @param <K> the key type
+     * @param <V> the value type
+     * @return the resolved property value as an instance of the requested Map (not {@code null})
+     * @throws IllegalArgumentException if a key or a value cannot be converted to the specified types
+     * @throws NoSuchElementException if no direct sub properties could be found.
+     */
+    @Experimental("Extension to retrieve mandatory sub properties as a Map")
+    public <K, V> Map<K, V> getMapValue(String name, Class<K> kClass, Class<V> vClass) {
```

Comment:
Wondering if we should just name this `getValue`. It shouldn't clash with any of the other methods and it will be consistent with `getValue` and `getValues`. What do you think?

Reply:
If so I would rather use `getValues` since in a `Map` you have more than one value, don't you agree?